src/components/Personalization/index.js

```diff
@@ -93,6 +93,18 @@ const createPersonalization = ({ config, logger, cookieJar }) => {
         ruleComponentModules = initRuleComponentModules(createCollect(collect));
       },
       onBeforeEvent(event, options, isViewStart) {
+        if (authoringMode) {
+          logger.warn("Rendering is disabled, authoring mode.");
+
+          event.mergeQuery({
+            personalization: {
+              enabled: false
+            }
+          });
+
+          return Promise.resolve();
```

Comment:
Feel free to leave this the way that it is, but it's also fine if you just want to `return` instead of returning a promise.

Reply:
@Aaronius I've tried that, but ESLint is complaining.